handler/src/main/java/io/netty/handler/ssl/JdkSslServerContext.java

```diff
@@ -80,6 +99,34 @@ public JdkSslServerContext(File certChainFile, File keyFile, String keyPassword)
      *                         {@code 0} to use the default value.
      * @param sessionTimeout the timeout for the cached SSL session objects, in seconds.
      *                       {@code 0} to use the default value.
+     * @param keyStore the used keystore type
+     * @deprecated use {@link SslContextBuilder}
+     */
+    @Deprecated
+    public JdkSslServerContext(
+            File certChainFile, File keyFile, String keyPassword,
+            Iterable<String> ciphers, Iterable<String> nextProtocols,
+            long sessionCacheSize, long sessionTimeout, String keyStore) throws SSLException {
```

Comment:
It makes no sense to add a new public method just to mark it `@deprecated`. I think you want to make it package-private and remove the `@Deprecated`